rafiki/container/container_manager.py

```diff
@@ -20,15 +20,18 @@
 import abc
 from typing import List, Dict, Tuple, Any
 
```

Comment:
can we select the container platform (docker swarm or k8s) when we deploy rafiki?
both can work?

Reply:
both can work